pkg/syncer/syncer_test.go

```diff
@@ -760,7 +777,10 @@ func runTestFullSyncWorkflows(t *testing.T) {
 		t.Fatal(err)
 	}
 	waitForListerSync()
-	csiFullSync(ctx, metadataSyncer)
+	err = CsiFullSync(ctx, metadataSyncer)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
I didn't find any new test to test full sync when the feature flag is enabled.  Can you add a new test?

Reply:
we don't have any test with feature flag enabled so far in our CSI. If needed i will add it as separate PR. 
We can get this in and will have that in subsequent PR.

Reply:
Sure, please submit a followup PR to add a unit test.